projects/exui-common-lib/src/lib/components/user-select/user-select.component.spec.ts

```diff
@@ -62,4 +62,15 @@ describe('UserSelectComponent', () => {
     });
     fixture.detectChanges();
   }));
+
+  it('should clear filter users when value is empty', async(() => {
```

Comment:
Do we need async here

Reply:
it was result of copy existing test and modifying it. async  is not required